<ui version="4.0" >
 <author>Copyright (C) 2006-2010 Felix Geyer</author>
 <class>ConfigErrorAlerts</class>
 <widget class="QWidget" name="ConfigErrorAlerts" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupSound" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="title" >
      <string>Alert sound</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkSoundPlay" >
        <property name="text" >
         <string>Play a sound when a download error occurs</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="1" column="1" >
         <widget class="QRadioButton" name="radioSoundCust" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Custom sound</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLineEdit" name="editSoundCust" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QRadioButton" name="radioSoundDef" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Default sound</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3" >
         <widget class="QPushButton" name="buttonSoundBrowse" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QPushButton" name="buttonSoundTest" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Test sound</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkSoundPlay</tabstop>
  <tabstop>radioSoundDef</tabstop>
  <tabstop>radioSoundCust</tabstop>
  <tabstop>editSoundCust</tabstop>
  <tabstop>buttonSoundBrowse</tabstop>
  <tabstop>buttonSoundTest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
